### PR TITLE
Cleaned up component and added attributes for control

### PR DESCRIPTION
**Pull request by AleksandrZhukov**
- Cleaned up component
- Added attributes for control
- Added demo

### CODE REVIEW

**Comment by jbueza** [comment]
Hey @AleksandrZhukov !

Thanks for the PR!  :+1:  :+1:

**Comment by AleksandrZhukov** [comment]
If you have any issues I can fix them too :smirk:

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "paper-vertical-tabs",
+  "version": "1.0.0",
+  "description": "paper vertical tabs",
+  "main": "paper-vertical-tabs.js",
+  "scripts": {
+    "start": "./node_modules/polyserve/bin/polyserve"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/AleksandrZhukov/paper-vertical-tabs.git"
```
Comment (jbueza) [inline package.json]: Hi @AleksandrZhukov  can you please change this to:

`"url": "git+https://github.com/Battlefy/paper-vertical-tabs.git"`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "bugs": {
+    "url": "https://github.com/AleksandrZhukov/paper-vertical-tabs/issues"
```
Comment (jbueza) [inline package.json]: Same with this please? :)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "paper-vertical-tabs",
+  "version": "1.0.0",
+  "description": "paper vertical tabs",
+  "main": "paper-vertical-tabs.js",
+  "scripts": {
+    "start": "./node_modules/polyserve/bin/polyserve"
+  },
```
Comment (jbueza) [inline package.json:11]: Please add yourself to the contributors list here

Comment (jbueza) [inline package.json:11]: http://browsenpm.org/package.json#contributors

**Comment by AleksandrZhukov** [comment]
Sorry I was in a hurry, because this component was needed for my project as soon as possible

**Comment by BelfordZ** [comment]
Looks good.

Thanks for contributing back, @AleksandrZhukov

**Pull request merged**